### DISCUSSION TITLE
Feedback/discussion of the new 5% flag treshold

### DISCUSSION THREAD

**Discussion by DandelionSprout** [uBlockOrigin/uBlock-issues#2947] [General]
Very recent precedence at https://github.com/EasyDutch-uBO/EasyDutch/issues/92#issuecomment-1799538277 and https://github.com/uBlockOrigin/uBlock-issues/issues/2946#issuecomment-1803660608 have established a "5% of all entries" treshold to have a flag added to them in assets.json, so I began thinking today: How would that treshold affect my lists:

For Faroe Islands, I landed on 34~36 entries. I'd personally call it impressive for an archipelago with only 55,000 residents, but I can see it'd fall short.

Much more worryingly, is how the treshold at face value would affect Iceland. Even when using very generous counting methods, the 5% treshold for Nordic Filters would be at 112 entries. Icelandic entries measure approx. 104. It is widely established beyond reasonable doubt that Iceland is a core focus of the list, so I can only really imagine the treshold would have to be changed.

How about something like "40 (or 50) entries or 3% of all entries, whichever is lowest"? It was the best idea I could come up with after waking up an hour ago, and would make things easier to calculate when considering generic hiding rules and such.

**Comment by MasterKia** [comment]
Much like AdGuard's filterlist inclusion, I think this threshold has been come up with on the fly.
So I doubt it will be acted upon as a set-in-stone rule.

> [The filter should be relatively popular, meaning:
    if there is a repository on GitHub, the number of stars should be at least 50](https://github.com/AdguardTeam/FiltersRegistry#what-filters-can-be-added-to-this-repository)

**Comment by ghost** [comment]
I vote for the sorting option.

# :christmas_tree:
